JongHo/app.js

```diff
@@ -29,10 +29,18 @@ app.use(cors());
 app.use(morgan("combined"));
 
 app.get("/ping", (request, response) => {
-  response.status(201).send("pong");
+  response.status(201).send("pog");
 });
+
 //create a user
-app.post("/user", (request, response) => {});
+app.post("/userCreate", async (request, response) => {
+  const { name, email, profile_image, password, age } = request.body;
```

Comment:
```suggestion
  const { name, email, profileImageUrl, password, age } = request.body;
```
- JavaScript에서는 기본적으로 `snake_case`가 아닌 `CamelCase`로 작성합니다.
- 또한 snake_case, CamelCase를 뒤죽박죽 섞어서 사용하면 안됩니다.
- 하나의 컨벤션으로 통일해서 작성해 주세요.